ICSSoft.STORMNET.Business/SQLDataService/SQLDataService.cs

```diff
@@ -4123,85 +4121,63 @@ private void AddOpertaionOnTable(StringCollection OpTables, SortedList TableOper
             LoadingCustomizationStruct cs = new LoadingCustomizationStruct(GetInstanceId());
 
             cs.Init(new ColumnsSortDef[0], func, new Type[] { view.DefineClassType }, view, new string[0]);
-            string sq = GenerateSQLSelect(cs, false);
-
-            if (sq != string.Empty)
+            object state = null;
+            BusinessServer[] bs = BusinessServerProvider.GetBusinessServer(view.DefineClassType, DataServiceObjectEvents.OnDeleteFromStorage, this);
+            if (bs != null && bs.Length > 0)
+            { // Если на детейловые объекты навешены бизнес-сервера, то тогда детейлы будут подгружены
```

Comment:
Комментарий в отдельной строке.
```suggestion
            {
              // Если на детейловые объекты навешены бизнес-сервера, то тогда детейлы будут подгружены.
```